websites/H/HBO Max/dist/metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://schemas.premid.app/metadata/1.0",
```

Comment:
Not required, but you might as well bump the schema while you're doing these changes `1.4` is latest - https://schemas.premid.app/metadata/1.4

Reply:
Bumped metadata version